roles/common/tasks/monitoring.yml

```diff
@@ -207,6 +207,11 @@
 - name: log scanning caching dir
   file: path=/var/cache/check-log owner=sensu state=directory
 
+- name: bad memory alert
+  sensu_check: name=bad-memory plugin=check-mcelog.sh use_sudo=true
+               auto_resolve=false interval=20 occurrences=1
+  notify: restart sensu-client
```

Comment:
Wouldn't this need the same conditional around ppc64 as the installation of mcelog itself?